source/dotnet/Test/AdaptiveCards.Templating.Test/TestTransform.cs

```diff
@@ -13123,8 +13126,8 @@ public void TestSerialization()
                 "\"body\": [" + 
                 "{" + 
                 "\"id\": \"stringWithJson\"," + 
-                "\"type\": \"TextBlock\"," + 
-                @"""text"": ""String With JSON - \""{\\\""number1\\\"":23}\""""," + 
+                "\"type\": \"TextBlock\"," +
+                @"""text"": ""String With JSON - \u0022{\\u0022number1\\u0022:23}\u0022""," + 
```

Comment:
I'm little bit concerned about this change.
will this text produce the correct AC? it's only comparing if two jsons are equal. Will .Net AC SDK render a correct AdaptiveCards?

Reply:
The change here is just that Newtonsoft was preserving escaped quotes in strings as `\"` which is not to spec. System.Text.Json is escaping these to \u0022. That's why this changed. Functionally it's the same.